### PR TITLE
chore: update i18next to v19.0.3

### PR DESCRIPTION
**Pull request by ratson**
Update to latest i18next

### CODE REVIEW

File: types.d.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
 } from 'react-i18next'
 import { LinkProps } from 'next/link'
 import { SingletonRouter } from 'next/router'
-import i18next from 'i18next'
+import * as i18next from 'i18next'
```
Comment (isaachinman) [inline types.d.ts]: This shouldn't be necessary with `esModuleInterop` set to `true`. Am I misunderstanding something?

Comment (ratson) [reply types.d.ts]: I got error without it

Comment (isaachinman) [reply types.d.ts]: We'll also need lockfile changes. I'll handle this upgrade separately. Thanks for bringing it to my attention.

**Comment by isaachinman** [comment]
Fixed by #608.

**Pull request closed without merging**